app/volume.go

```diff
@@ -192,24 +205,26 @@ func (job *Job) handleVolumeDetachment() {
 			// !volume.DisableFrontend condition makes sure that volume is detached by this recurring job,
 			// not by the auto-reattachment feature.
 			if volume.State == string(types.VolumeStateDetached) && !volume.DisableFrontend {
-				logrus.Infof("Volume %v is detached", volumeName)
+				job.logger.Infof("Volume %v is detached", volumeName)
 				return
 			}
 			if volume.State == string(types.VolumeStateAttached) {
-				logrus.Infof("Attempting to detach volume %v from all nodes", volumeName)
+				job.logger.Infof("Attempting to detach volume %v from all nodes", volumeName)
 				if _, err := volumeAPI.ActionDetach(volume, &longhornclient.DetachInput{HostId: ""}); err != nil {
-					logrus.Infof("%v ", err)
+					job.logger.Infof("%v ", err)
 				}
 			}
 
 		} else {
-			logrus.Infof("%v ", err)
+			job.logger.Infof("%v ", err)
```

Comment:
consider adding a log message describing what failed then have the error in a withError field.

Reply:
Thank you. Fixed